client/lib/perfmon/index.js

```diff
@@ -173,7 +173,7 @@ module.exports = function() {
 	if ( !initialized ) {
 		var MutationObserver = window.MutationObserver || window.WebKitMutationObserver;
 
-		if ( MutationObserver ) {
+		if ( MutationObserver && window.performance ) {
```

Comment:
I'm not sure we need to abort here just if `performance.now()` isn't available, unless you are worried about the edge cases, which could make sense